features/cellular/framework/API/CellularNetwork.h

```diff
@@ -55,6 +55,7 @@ class CellularNetwork : public NetworkInterface {
                                             optimization. */
         SUPPORTED_UE_OPT_MAX
     };
+    typedef Supported_UE_Opt Supported_Network_Opt;
```

Comment:
What about one type which doesnt have UE or network in the name of the values and it could be used bithe as UE and network values? Like:
enum CIoT_Opt {
CIOT_OPT_NO_SUPPORT
CIOT_OPT_CONTROL_PLANE
....